pages/common/composer.md

```diff
@@ -0,0 +1,31 @@
+# composer
+
+> Dependency Manager for PHP.
+
+- Interactively create a composer.json file:
+
+`composer init`
+
+- Create new project from an existing package:
+
+`composer create-project {{project_name}} {{target_path}} {{version}}`
+
+- Install packages specified in composer.json file:
```

Comment:
Yeah, it's best to be explicit than to assume prior knowledge. We could use something based on what the Composer website writes in the [intro page](https://getcomposer.org/doc/00-intro.md): "[composer manages dependencies] on a per-project basis, installing them in a directory inside your project. By default it does not install anything globally."